browsers/safari_ios.json

```diff
@@ -158,8 +158,18 @@
           "engine": "WebKit",
           "engine_version": "610.1.28"
         },
+        "14.3": {
```

Comment:
This "14.3" entry can be removed if it ends up not being used as a version anywhere in the data.

If I've misunderstood and you do want "14.3" for iOS but not "14.0.2" for macOS, then that seems OK too, but a lot more changes would be needed to get the data right.

Reply:
Also addressed in a086511.